projects/plugins/jetpack/extensions/shared/plan-utils.js

```diff
@@ -31,7 +31,12 @@ import { requiresPaidPlan } from './register-jetpack-block';
  */
 export function getUpgradeUrl( { planSlug, plan, postId, postType } ) {
 	// WP.com plan objects have a dedicated `path_slug` field, Jetpack plan objects don't.
-	const planPathSlug = startsWith( planSlug, 'jetpack_' ) ? planSlug : get( plan, [ 'path_slug' ] );
+	let planPathSlug = startsWith( planSlug, 'jetpack_' ) ? planSlug : get( plan, [ 'path_slug' ] );
+
+	// Overrites path to `business` for Atomic sites.
```

Comment:
Super minor, but there's a typo:

```suggestion
	// Overwrites path to `business` for Atomic sites.
```